pkg/flist/flist.go

```diff
@@ -576,4 +587,16 @@ func waitMountedLog(timeout time.Duration, logfile string) error {
 	return <-cErr
 }
 
+func forceStop(pid int) error {
+	log.Info().Int("pid", pid).Msg("trying to force stop by killing the process")
+	p, err := os.FindProcess(int(pid))
+	if err != nil {
+		return err
+	}
+	if err := p.Signal(os.Kill); err != nil {
```

Comment:
the main 0-fs mount point is a "unionfs" if it some how was mounted without 0-fs detect that, then it's better to first ask it nicely "SIGTERM" before trying to kill it. This way 0-fs can try to unmount the "read-only" fuse layer, before shutting down. it still not dead by then, we can do sigkill.